### PR TITLE
[new] [author] particles.js

### PR DESCRIPTION
**Pull request by VincentGarreau**
Adding particles.js https://github.com/VincentGarreau/particles.js

### CODE REVIEW

File: ajax/libs/particles.js/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "filename": "particles.min.js",
+  "version": "2.0.0",
+  "description": "A lightweight JavaScript library for creating particles.",
+  "homepage": "http://vincentgarreau.com/particles.js",
+  "authors": [
+    "Vincent Garreau <vin.garreau@gmail.com>"
+  ],
+  "keywords": [
+    "particle",
+    "particles"
```
Comment (PeterDaveHello) [inline ajax/libs/particles.js/package.json]: missing `,` here

**Comment by VincentGarreau** [comment]
Thank you @PeterDaveHello, fixed :)

**Comment by PeterDaveHello** [comment]
Would you please squash these commits into only one commit as our document said? thanks!

**Comment by drewfreyling** [comment]
Closing due to no response

**Comment by PeterDaveHello** [comment]
ping @VincentGarreau

**Comment by Piicksarn** [comment]
ping @VincentGarreau 
@cdnjs/intern @cdnjs/intern2

**Comment by pvnr0082t** [comment]
`particles.js` had already been hosted via 8b5007b888729ed6840a1a81fd3384a11971cb02. I'm going to close this PR. @VincentGarreau Thanks for your contribution.

**Comment by PeterDaveHello** [comment]
@pvnr0082t please mention the actually commit and PR to add this lib next time, you mentioned a merge commit.

**Pull request closed without merging**